BlockV/Face/Face Views/Layered Image/LayeredImageFaceView.swift

```diff
@@ -0,0 +1,260 @@
+        ///
+        /// The config has a set of default values. If the face config section is present, those values are used in
+        /// place of the default ones.
+        ///
+        /// ### Legacy Support
+        /// The first resource name in the resources array (if present) is used in place of the activate image.
+        init(_ faceModel: FaceModel) {
+            // legacy: overwrite fallback if needed
+            self.imageName ?= faceModel.properties.resources.first
+
+            if let config = faceModel.properties.config {
+                // assign iff not nil
+                if let scaleString = config["scale"]?.stringValue {
+                    self.scale ?= Config.Scale(rawValue: scaleString)
+                }
+                self.imageName ?= config["name"]?.stringValue
+            }
+        }
+    }
+
+    private let config: Config
+
+    // MARK: - Init
+    required init(vatom: VatomModel, faceModel: FaceModel) {
+        /// init face config
+        self.config = Config(faceModel)
+        super.init(vatom: vatom, faceModel: faceModel)
+
+		///ensure base has correct bounds with the 'parent' vAtom
+		baseLayer.frame = self.bounds
+		baseLayer.vatom = vatom
+        self.addSubview(baseLayer)
+		
+		/*
+		BLCOKv.
+		*/
+		self.vAtomStateChanged()
+		
+		/// listen to websocket for state changes on LayeredImageView
+		BLOCKv.socket.onVatomStateUpdate.subscribe(with: self) { ( _ stateUpdateEvent) in
+			self.vAtomStateChanged()
+		} 
+    }
+
+    required public init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) should not be called on Face Views. Please use VatomView.")
+    }
+
+	// MARK: - View Lifecylce
+
+	override func layoutSubviews() {
+		super.layoutSubviews()
+		updateContentMode(forLayer: baseLayer)
+	}
+
+	private func updateContentMode(forLayer: Layer) {
+		/// check face config
+		switch config.scale {
+		case .fill: forLayer.contentMode = .scaleAspectFill
+		case .fit:  forLayer.contentMode = .scaleAspectFit
+		}
+	}
+
+    // MARK: - Face View Lifecycle
+
+    /// Begin loading the face view's content.
+    func load(completion: ((Error?) -> Void)?) {
+		updateResources(completion: completion)
+    }
+	
+    func vatomChanged(_ vatom: VatomModel) {
+		self.vatom = vatom
+		updateResources(completion: nil)
+    }
+	
+    func unload() {
+		self.baseLayer.image = nil
+    }
+
+	// MARK: - Resources
+
+	private func updateResources(completion: ((Error?) -> Void)?) {
+
+		/// extract resource model
+		guard let resourceModel = vatom.props.resources.first(where: { $0.name == config.imageName }) else {
+			return
+		}
+
+		/// encode url
+		guard let encodeURL = try? BLOCKv.encodeURL(resourceModel.url) else {
+			return
+		}
+
+		/// load image (automatically handles reuse)
+		Nuke.loadImage(with: encodeURL, into: self.baseLayer) { (_, error) in
+			self.isLoaded = true
+			completion?(error)
+		}
+	}
+	
+	///Fetch the children of the given vAtom and apply the relevant layers
```

Comment:
Please ensure a leading space and add grammar, e.g. fullstop, when documenting methods.
```suggestion
	/// Fetch the children for the specified vAtom identifier and apply the relevant layers.
```